web/docs/operators/chart.md

```diff
@@ -45,6 +45,18 @@ Control how the values are grouped when rendered as a chart.
 Possible values are `grouped` and `stacked`.
 Defaults to `grouped`.
 
+### `x-axis-type=<x-axis-type>` (`line`, `area`, and `bar` charts only)
+
+Sets the x-axis scale type.
+Possible values are `linear` and `log`.
+Defaults to `linear`.
```

Comment:
```suggestion
Possible values are `linear` and `log`.

Defaults to `linear`.
```

Reply:
The same style has too many other instances (without the newline) in the file and I don't feel like it adds much value.

Reply:
Can you fix all of them. All other operators have that convention about putting the defaults into a dedicated line.

Reply:
Okay done.